### DISCUSSION TITLE
install package ifcopenshell in miniconda

### DISCUSSION THREAD

**Discussion by StefanSutter** [IfcOpenShell/IfcOpenShell#2455] [Q&A] [answered]
Hi everyone
I own a new MB Air with an M2. For several weeks I tried to install ifcopenshell in a miniconda environment. Initially there was no package available on anaconda so I tried to compile it. Several attempts with the help of forums always lead to an error on linking the python wrapper: File not found <Python.h> although I had in my opinion a correct path included in the cmake (DPYTHON_INCLUDE_DIR=/opt/minconda3/envs/myenvironmentname/include/python3.9). So I gave up und decided to wait and see if there will be some compiled package, as I could no longer help myself and I did not find any new hints.

Finally I found a package in anaconda (https://anaconda.org/conda-forge/ifcopenshell) and I thought: "finally!!!"
But it does not work. It seems, that the dylibs will be installed, but there is no python wrapper in site-packages. And thus an "import ifcopenshell" will simply fail. 

Has anyone any hint what I could do? It seems really strange as the several versions have been downloaded by really many users and I simply cannot imagine that I am the only one having difficulties?
Thanks in advance

**Comment by Moult** [comment, marked as answer]
I think that the Conda builds do not support Apple M1 yet. However you can always grab the latest build from here: https://blenderbim.org/docs-python/ifcopenshell-python/installation.html

Note that the package in Anaconda should be installed with `conda install -c ifcopenshell -c conda-forge ifcopenshell`. Check the link above for full instructions.

**Reply by StefanSutter** [reply]
Many many thanks. "Read the docs" would have helped...
the conda install did not work directly but downloading the package for m1 "by hand" and copying its contents to site-packages really helped. First tests are successful!

**Reply by Moult** [reply]
Docs have now been updated to say how Anaconda doesn't host M1 and to use the pre-built packages instead.
